File: app/src/androidTest/java/io/github/getsixtyfour/openpyn/FileWritingScreenshotCustomCallback.java
package io.github.getsixtyfour.openpyn;

import android.content.Context;

import java.io.File;

import org.jetbrains.annotations.NonNls;

import tools.fastlane.screengrab.FileWritingScreenshotCallback;

public class FileWritingScreenshotCustomCallback extends FileWritingScreenshotCallback {

    public FileWritingScreenshotCustomCallback(Context appContext) {
        super(appContext);
    }

    @Override
    protected File getScreenshotFile(File screenshotDirectory, String screenshotName) {
        @NonNls String screenshotFileName = screenshotName + ".png";
        return new File(screenshotDirectory, screenshotFileName);
    }
}

File: app/src/androidTest/java/io/github/getsixtyfour/openpyn/FileWritingScreenshotCustomCallback.kt
package io.github.getsixtyfour.openpyn

import android.content.Context
import tools.fastlane.screengrab.FileWritingScreenshotCallback
import java.io.File

class FileWritingScreenshotCustomCallback(appContext: Context, locale: String) : FileWritingScreenshotCallback(appContext, locale) {

    override fun getScreenshotFile(screenshotDirectory: File, screenshotName: String): File {
        val screenshotFileName = "$screenshotName.png"
        return File(screenshotDirectory, screenshotFileName)
    }
}
